recipes/dotnet_agent.rb

```diff
@@ -68,6 +68,7 @@
     :controller_accesskey => controller['accesskey'],
     :proxy_host => proxy['host'],
     :proxy_port => proxy['port'],
+	:standalone_apps => agent['standalone_apps']
```

Comment:
I'm not sure why the indentations are getting messed up. This happens only when I publish it to github. My local files looks okay.